plugins/inputs/system/DISK_README.md

```diff
@@ -94,21 +131,27 @@ Data collection is based on github.com/shirou/gopsutil. This package handles pla
 
 ### Sample Queries:
 
-Calculate percent IO utilization per disk and host:
+#### Calculate percent IO utilization per disk and host:
 ```
 SELECT derivative(last("io_time"),1ms) FROM "diskio" WHERE time > now() - 30m GROUP BY "host","name",time(60s)
 ```
 
+#### Calculate average queue depth:
+`iops_in_progress` will give you an instantaneous value. This will give you the average between polling intervals.
+```
+SELECT derivative(last("weighted_io_time",1ms))/1000 from "diskio" WHERE time > now() - 30m GROUP BY "host","name",time(60s)
```

Comment:
This is the same equation as used by `iostat`: https://github.com/sysstat/sysstat/blob/33557c3db463ac6efb337a67dd5f099609d62b30/iostat.c#L1042:L1044